Source/missiles.cpp

```diff
@@ -1094,21 +1095,25 @@ void AddLightningRune(Missile &missile, const AddMissileParameter &parameter)
 	int dmg = 16 * (GenerateRndSum(10, 2) + lvl + 2);
 	missile._midam = dmg;
 	AddRune(missile, parameter.dst, MIS_LIGHTWALL);
+	UseMana(Players[missile._misource], SPL_RUNELIGHT);

```

Comment:
Runes don't use mana

Reply:
The function `UseMana` is also called (in master) with `_pSplType` set to `RSPLTYPE_SCROLL` (`RemoveCurrentSpellScroll`) and `RSPLTYPE_CHARGES` (`UseStaffCharge`).
I reused it for runes, cause they are also considered "scrolls". 😕 

But yes the name is misleading (in master and in this pr even more).
Do you have a idea for a new name?

Reply:
Ok i thought that might be the case. How about `ConsumeSpell()`?

Reply:
👍 
I changed it to `ConsumeSpell` 🙂